ample/modelling/rosetta_model.py

```diff
@@ -611,17 +745,16 @@ def mr_cmd(self, template,alignment, nstruct,seed):
         cmd = self.cmd_add_restraints(cmd)
         return cmd
     
-    def nmr_remodel(self, models, ntimes=None, alignment_file=None, remodel_fasta=None):
-        if remodel_fasta:
-            assert os.path.isfile(remodel_fasta), "Cannot find remodel_fasta: {0}".format(remodel_fasta)
-        if ntimes:
-            assert isinstance(ntimes, int), "ntimes is not an int: {0}".format(ntimes)
+    def do_nmr_remodel(self, models):
+        if self.mnr_remodel_fasta and not os.path.isfile(self.mnr_remodel_fasta):
+                raise RuntimeError("Cannot find remodel_fasta: {0}".format(self.mnr_remodel_fasta))
+        if self.nmr_process_ntimes:
+            assert isinstance(self.nmr_process_ntimes, int), "ntimes is not an int: {0}".format(self.nmr_process_ntimes)
```

Comment:
feel like this should be an exception and the assert statement part of the `if`

Reply:
agreed